static/app/components/globalModal/index.tsx

```diff
@@ -81,7 +81,7 @@ type Props = {
   onClose?: () => void;
 };
 
-const getModalPortal = memoize(() => {
+export const getModalPortal = memoize(() => {
```

Comment:
Can we move this to a new file? I think this will get confusing when we have multiple exports (outside of like, utils, or styled components).

Reply:
@billyvg So have a new file just to export `getModalPortal`?

Reply:
Yeah, we generally stick to 1 export per file besides the above mentioned exceptions (and sometimes we'll export the styled component version alongside the component). It makes things easier to find.